src/dataExplorer/components/DataExplorerPage.tsx

```diff
@@ -114,10 +119,13 @@ const DataExplorerPage: FC = () => {
   const {flowsCTA, scriptQueryBuilder, setFlowsCTA} =
     useContext(AppSettingContext)
   useLoadTimeReporting('DataExplorerPage load start')
-  const showNewExplorer = scriptQueryBuilder && isFlagEnabled('newDataExplorer')
   const history = useHistory()
-  const isNewIOxOrg = useSelector(selectIsNewIOxOrg)
+  const isNewIOxOrg =
+    useSelector(selectIsNewIOxOrg) &&
+    !isFlagEnabled('showOldDataExplorerInNewIOx')
   const showNotebooks = useSelector(selectShouldShowNotebooks)
+  const showNewExplorer =
```

Comment:
This is another example where it would be easy to confuse this boolean expression with a method name